test-projects/debug-log/worker.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2019 Adobe Inc. All rights reserved.
```

Comment:
```suggestion
 * Copyright 2021 Adobe Inc. All rights reserved.
```

Reply:
Well it's a slight copy of an existing test file :)